addon/adapters/json-api.js

```diff
@@ -109,14 +246,6 @@ var JSONAPIAdapter = RESTAdapter.extend({
   */
   coalesceFindRequests: false,
 
-  /**
-    @method findMany
-    @param {DS.Store} store
-    @param {DS.Model} type
-    @param {Array} ids
-    @param {Array} snapshots
-    @return {Promise} promise
-  */
```

Comment:
The RESTAdapter has better docs for these apis and these minimal doc blocks in the JSONAPIAdapter are preventing the better docs from being rendered. They will still be listed as public on the doc page.